neqo-transport/src/connection/tests/migration.rs

```diff
@@ -969,6 +971,21 @@ fn migration_invalid_state() {
         .is_err());
 }
 
+#[test]
+fn migration_disabled() {
+    let mut client = default_client();
+    let mut server = new_server(ConnectionParameters::default().disable_migration(true));
+    connect(&mut client, &mut server);
+    assert_eq!(client.state(), &State::Confirmed);
+    assert_eq!(server.state(), &State::Confirmed);
```

Comment:
Isn't this already checked by `connect` above?

https://github.com/mozilla/neqo/blob/a4692b57374b46969714269b1ae1d216903b507a/neqo-transport/src/connection/tests/mod.rs#L275-L276

Reply:
Possibly. Copied it from another test.